GTMAdminOps/about.rst

```diff
@@ -5,16 +5,16 @@
    Page ###Page### of ###Total###
 
 .. index::
-    About YottaDB/GT.M
+    About YottaDB
```

Comment:
I think we should retain YottaDB/GT.M for now, at least until we convert all the examples to YottaDB examples.